.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Test
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest ]
+    runs-on: ${{ matrix.os }}
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - uses: actions/cache@v2
+        with:
+          path: Whist-Desktop/Librarys
+          key: Library-Whist-Desktop-${{ matrix.os }}
+          restore-keys: |
+            Library-MyProjectName-
```

Comment:
```suggestion
          path: Whist-Desktop/Library
          key: Library-Whist-Desktop-${{ runner.os }}
          restore-keys: |
            Library-Whist-Desktop-
```

Reply:
Dang copy paste, but why use runner here?

Reply:
its safer, maybe we want to remove the matrix, then this still works

Reply:
Ah didn't know this will work too